### PR TITLE
Prepare for publishing nydus-storage and nydus-rafs 

### PR DESCRIPTION
**Pull request by jiangliu**
Prepare for publishing nydus-storage and nydus-rafs crates

### CODE REVIEW

File: storage/Cargo.toml [hunk 1]
```diff
@@ -12,8 +15,8 @@ arc-swap = "=0.4"
 base64 = { version = "0.13.0", optional = true }
 bitflags = "1.2.1"
 futures = "0.3"
-governor = "0.4"
-hmac = { version = "0.8.1", optional = true }
+governor = "=0.4.1"
```
Comment (wllenyj) [inline storage/Cargo.toml:18]: Sure to use `=` ?

Comment (jiangliu) [reply storage/Cargo.toml:18]: Add comments for it
"# pin governor to avoid multi versions of hashbrown"

File: Cargo.toml [hunk 2]
```diff
@@ -59,7 +59,7 @@ nydus-app = { path = "app" }
 nydus-error = { path = "error" }
 nydus-utils = { path = "utils" }
 rafs = { path = "rafs", features = ["backend-registry", "backend-oss"] }
-storage = { path = "storage" }
+nydus-storage = { version = "0.5.0", path = "storage" }
```
Comment (imeoer) [inline Cargo.toml:62]: The commit message should be `"storage: release v0.5.0"` ?

File: error/Cargo.toml [hunk 3]
```diff
@@ -15,3 +15,7 @@ libc = "0.2"
 log = "0.4"
 serde = { version = "1.0.110", features = ["serde_derive", "rc"] }
 serde_json = "1.0.53"
+
+[package.metadata.docs.rs]
+all-features = true
+targets = ["x86_64-unknown-linux-gnu", "x86_64-apple-darwin"]
```
Comment (imeoer) [inline error/Cargo.toml:21]: Does this affect `aarch64-unknown-linux-gnu`?

Comment (jiangliu) [reply error/Cargo.toml:21]: good point, will add it

**Review by imeoer — APPROVED** [review]

File: error/Cargo.toml (hunk 3 above)
Comment (jiangliu) [reply error/Cargo.toml]: Fixed

File: Cargo.toml (hunk 2 above)
Comment (jiangliu) [reply Cargo.toml:62]: Fixed

**Pull request merged**